src/ImageRunModal.jsx

```diff
@@ -378,10 +378,20 @@ export class ImageRunModal extends React.Component {
                         client.postContainer(isSystem, "start", reply.Id, {})
                                 .then(() => this.props.close())
                                 .catch(ex => {
-                                    this.setState({
-                                        dialogError: _("Container failed to be started"),
-                                        dialogErrorDetail: cockpit.format("$0: $1", ex.reason, ex.message)
-                                    });
+                                    // If container failed to start remove it, so a user can fix the settings and retry and
+                                    // won't get another error that the container name is already taken.
+                                    client.delContainer(isSystem, reply.Id, true).then(() => {
```

Comment:
Please move the .then() to the next line. It should (1) be on the same intentation as the corresponding .catch(), and it's hard to read these chained calls.

Reply:
Done